src/test/java/gov/va/viers/cdi/emis/MilitaryInfoTests.java

```diff
@@ -34,12 +34,12 @@
 
   @Autowired private DoDAdapterClient dodClient;
 
-  public void initSuccessMock() {
+  private void initMock(String sampleResponsePath, String edipi) {
     EMISmilitaryServiceEligibilityResponseType dodResponse;
     try {
       // Grabbing from classpath
       ClassPathResource sampleResponse =
-          new ClassPathResource("/vadirResponses/exampleSuccessVadirResponse_MilInfoEligSvc.xml");
+          new ClassPathResource("/vadirResponses/" + sampleResponsePath);
```

Comment:
I like this a lot. The only thing I'd want is to update the debug logger to print out the sampleResponsePath since it's dynamic now.